.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
don't commit your vscode settings in the future. try this to remove it: https://stackoverflow.com/questions/18357511/git-remove-committed-file-after-push